### DISCUSSION TITLE
Reversion Mode?

### DISCUSSION THREAD

**Discussion by JohnMacrae** [celsworth/lxp-bridge#156] [Ideas]
I'm opening this as a discussion issue as I'm not aware of anywhere we can otherwise discuss.

Whilst my automation runs nicely from Node Red, I occasionally override it to take advantage of opportunities such as Plunge Pricing and Saving Sessions. I've been away from home and have been using VPN for these interventions however, in the middle of a recent session, the router died and could have left me with some expensive import sessions programmed into the inverter. I was wondering whether a 'heartbeat and revert mode' might be a useful safety measure should no access to the internet/pricing occur. I realise its an edge case but others might find it comforting to have.

**Comment by celsworth** [comment]
I've opened the Github Discussions feature and converted this one, congratulations on being our first discussion :D

As to the feature itself, might need a bit more fleshing out of the idea - what would it revert to? Some preprogrammed register values?

**Reply by JohnMacrae** [reply]
Thanks Chris, and thanks for your work on this. 
Regarding reversion - you could either choose some pre-programmed values (a simple json file?) or just clear all scheduled imports and exports so that at least you don't get caught out. I envisage a 'midnight reset' unless a heartbeat had been received within ???
